packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -101,89 +91,67 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
                         touched,
                     }) => (
                         <Form>
-                            {!selected_residence || !is_same_citizenship_modal ? (
-                                <ResidenceForm
-                                    header_text={localize('Thanks for verifying your email')}
-                                    class_prefix='account-signup'
+                            {!is_pasword_modal ? (
+                                <div className='account-signup__main'>
+                                    <Text as='h1' weight='bold' className='account-signup__heading'>
+                                        {localize('Select your country and citizenship:')}
+                                    </Text>
+                                    <ResidenceForm
+                                        class_prefix='account-signup'
+                                        errors={errors}
+                                        touched={touched}
+                                        setFieldTouched={setFieldTouched}
+                                        setFieldValue={setFieldValue}
+                                        residence_list={residence_list}
+                                        default_value={country}
+                                        history_value={history_value.current}
+                                    />
+                                    <CitizenshipForm
+                                        class_prefix='account-signup'
+                                        errors={errors}
+                                        touched={touched}
+                                        setFieldTouched={setFieldTouched}
+                                        setFieldValue={setFieldValue}
+                                        citizenship_list={residence_list}
+                                    />
+                                    <div className='account-signup__footer'>
+                                        <Button
+                                            className={classNames('account-signup__btn', {
+                                                'account-signup__btn--disabled':
+                                                    !values.residence ||
+                                                    !!errors.residence ||
+                                                    !values.citizenship ||
+                                                    !!errors.citizenship,
```

Comment:
```suggestion
                                                    !values.residence ||
                                                    !!errors.residence ||
                                                    !values.citizenship ||
                                                    !!errors.citizenship,
```
put it in a variable so that this can be reused. you can change the syntax from `=> (` to  `=> {` and return the JSX Element so that you can declare a local variable here